libs/common-services/src/main/java/uk/gov/di/ipv/core/library/persistence/item/IpvSessionItem.java

```diff
@@ -65,13 +65,14 @@ public void addVcReceivedThisSession(String vc) {
         vcReceivedThisSession.add(vc);
     }
 
-    public List<String> getFeatureSet() {
-        return (featureSet != null)
-                ? Stream.of(featureSet.split(",")).toList()
+    public List<String> getFeatureSetAsList() {
+        return (featureSet != null && !featureSet.equals("[ { \"S\" : \"\" } ]"))
```

Comment:
(If this doesn't work, then I think we can probably be a bit more restrictive and use the same regex as the FE to discard invalid featureset configurations)